### ISSUE TITLE
Suggestion: Add the icons to the power plan picker

### ISSUE THREAD

**Issue by FSund** [MetaFight/PowerBuddy#11]
Amazing app, I've been using it for several years, on both Windows 7, 8 and 10.

The colored icons improve the app a lot, but I think you should consider adding the icons to the power plan picker. This would add a visual clue in addition to the text.

Something like this (with different colors for each icon):
![image](https://cloud.githubusercontent.com/assets/3352483/24597773/099f8a92-1848-11e7-87df-ecc2b1c4152b.png)

Perhaps the default power plans should get some default (different) colors as well (this is a separate issue though).

**Comment by MetaFight** [comment]
Hrm, interesting idea.  I'm not sure putting the round icon next to the round radio button is the best UX though.

Also, at that scale, the icon is already pretty hard to parse visually.

How about showing the icon's colours instead?

**Comment by FSund** [comment]
Yeah, just indicating the color is a good idea. Perhaps just coloring the radio button is a solution?
